packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/pages/delete/pages/[...slug].astro

```diff
@@ -123,6 +151,8 @@ const pageTitle = `Delete Page | ${contextConfig.title}`;
                     class="border-solid border-2 border-gray-300 rounded-lg p-2 w-96"
                     placeholder="Your Post Slug (THIS WILL DELETE THE POST)"
                     required><br><br>
```

Comment:
```suggestion
                    required>
```

It's a pet peeve of mine to use `br` tags for positioning/styling 😂 just use a css margin